src/domain/login/LoginViewModel.ts

```diff
@@ -181,9 +223,9 @@ export class LoginViewModel extends ViewModel {
             }
         }
         this._abortHomeserverQueryTimeout = this.disposeTracked(this._abortHomeserverQueryTimeout);
-        this.queryHomeserver();
+        await this.queryHomeserver();
```

Comment:
Why `await` here and in other places? The intent is to not block further execution unless there's a reason to.

Reply:
I wasn't thinking clearly, removed these in 40276298041a70058f5447795d591f48af83ac96